addons/website/views/snippets.xml

```diff
@@ -79,17 +79,10 @@
         <div class="container">
             <div class="row s_nb_column_fixed">
                 <div class="col-lg-7 bg-white jumbotron rounded pt32 pb32" data-name="Box">
-                    <div class="row">
-                        <div class="col-lg-12 s_title s_col_no_bgcolor" data-name="Title">
-                            <h1 class="s_title_thin"><font style="font-size: 62px;"><b>Sell Online.</b> Easily.</font></h1>
-                        </div>
-                        <div class="col-lg-12 pt8 pb32 s_col_no_bgcolor" data-name="Text">
-                            <p class="lead">This is a simple hero unit, a simple jumbotron-style component <br/>for calling extra attention to featured content or information.</p>
-                        </div>
-                        <div class="col-lg-12 s_btn text-left pt16 pb16" data-name="Button">
-                            <a href="#" class="btn btn-epsilon rounded-circle">Do something</a>
-                        </div>
-                    </div>
+                    <h1 class="s_title_thin"><font style="font-size: 62px;">Sell Online. Easily.</font></h1>
+                    <p class="lead">This is a simple hero unit, a simple jumbotron-style component <br/>for calling extra attention to featured content or information.</p>
+                    <p class="lead"><br/></p>
```

Comment:
What's this ? Can't you put the `<br/>` in the above `<p/>` if it is to solve a bug ?

Reply:
This is not to fix a bug per se, it's to match the original layout without the use of padding (which the user would not be able to edit)

Because the p tag takes up no space, it's actually the br tag that's targetable by the editor, if it's outside of the p, the user can write directly into the enclosing div and the newly inserted text doesn't benefit from the "lead" class formatting. Much like when I replaced the font tag for headings with inline style, this is what the editor produces if you were to press enter while editing the text above.